### PR TITLE
Updates for GitHub's design refresh

### PR DESCRIPTION
**Pull request by madhavarshney**
Solves #31.

<p>
Screen Shot 2020-06-23 at 9 56 51 AM
Screen Shot 2020-06-23 at 9 56 21 AM
</p>

@brumm I'm opening this for you to take a look and take over 😄. Just gonna be a bit busy this week so I thought you can take over if you'd like to get this in earlier.

### CODE REVIEW

File: src/components/Node.js [hunk 1]
```diff
@@ -118,7 +118,7 @@ const Node = ({ type, name, path, parentCommitmessage, level }) => {
           style={{
             paddingLeft: INDENT_SIZE * level + 10,
             display: 'grid',
-            gridTemplateColumns: '18px 18px 1fr',
+            gridTemplateColumns: '26px 26px 1fr',
```
Comment (madhavarshney) [inline src/components/Node.js:121]: Feel free to play with these numbers to match the design refresh; I didn't spend much time on this.

File: src/components/RepoFileTree.js [hunk 2]
```diff
@@ -23,7 +20,7 @@ const RepoFileTree = () => {
           <Cell
             css={{
               display: 'inline-grid',
-              gridTemplateColumns: '18px 18px 1fr',
+              gridTemplateColumns: '22px 22px 1fr',
```
Comment (madhavarshney) [inline src/components/RepoFileTree.js:23]: Not sure if this actually did anything... I did this in the middle of the night and didn't actually check 😅. Feel free to revert this.

File: src/components/icons.js [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
       fillRule="evenodd"
-      d="M13 4H7V3c0-.66-.31-1-1-1H1c-.55 0-1 .45-1 1v10c0 .55.45 1 1 1h12c.55 0 1-.45 1-1V5c0-.55-.45-1-1-1zM6 4H1V3h5v1z"
+      d="M1.75 1A1.75 1.75 0 000 2.75v10.5C0 14.216.784 15 1.75 15h12.5A1.75 1.75 0 0016 13.25v-8.5A1.75 1.75 0 0014.25 3h-6.5a.25.25 0 01-.2-.1l-.9-1.2c-.33-.44-.85-.7-1.4-.7h-3.5z"
     />
   </svg>
 )
 
 export const FileIcon = props => (
-  <svg viewBox="0 0 12 16" width="100%" height="16" {...props}>
+  <svg viewBox="0 0 16 16" width="100%" height="16" color="#6a737d" {...props}>
```
Comment (madhavarshney) [inline src/components/icons.js:14]: Icons have been updated, colors changed.

File: src/constants.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-export const MOUNT_SELECTOR = 'table.files'
+export const MOUNT_SELECTOR = '.Details [role="grid"]'
```
Comment (madhavarshney) [inline src/constants.js:1]: We may need to figure out a better selector or more robust selector for this. For now this seems to have worked for the (minimum) pages on GitHub that I navigated.

**Comment by madhavarshney** [comment]
Also, I just noticed that the design refresh does not have borders in the file list. Maybe that should be updated?

**Comment by brumm** [comment]
Hey, I sorta did this in b120d680394102ed702446de4ffd668d22314a2e, could you give this a try?

**Pull request closed without merging**